tests/trestle/core/utils_test.py

```diff
@@ -44,3 +47,41 @@ def test_get_elements():
     assert (len(control_list) >= 1)
     group_list = mutils.get_elements_of_model_type(good_sample, catalog.Group)
     assert (len(group_list) >= 2)
+
+
+def test_has_no_duplicate_values_generic():
+    """Test presence of duplicate uuid."""
+    # test with pydantic catalog
+    cat = load_good_catalog()
+    assert mutils.has_no_duplicate_values_generic(cat, 'uuid')
+
+    yaml_path = pathlib.Path('tests/data/yaml')
+
+    # test with valid pydantic target
+    good_target_path = yaml_path / 'good_target.yaml'
+    good_target = ostarget.TargetDefinition.oscal_read(good_target_path)
+    assert mutils.has_no_duplicate_values_generic(good_target, 'uuid')
+
+    # test with pydantic target containing duplicates
+    bad_target_path = yaml_path / 'bad_target.yaml'
```

Comment:
I wonder whether we should make our 'bad_object' files describe why they are bad?